py/torch_tensorrt/dynamo/conversion/impl/split.py

```diff
@@ -0,0 +1,81 @@
+from typing import Any, Dict, List, Optional, Sequence, Tuple, Union, cast
```

Comment:
Minor point, but the linting may complain about unused import for `cast` here

Reply:
Removed it